src/Engine/ProtoCore/Utils/MathUtils.cs

```diff
@@ -4,30 +4,16 @@ namespace ProtoCore.Utils
 {
     public static class MathUtils
     {
-        public static double Tolerance = 1e-5; 
+        public static double Tolerance = 1e-5;
 
-        public static bool IsLessThan(double lhs, double rhs)
```

Comment:
@aparajit-pratap unfortunately I think we will only be able to obsolete these, not sure we can remove them now.

Reply:
Of course. Will make the necessary changes.